twindb_backup/modifiers/base.py

```diff
@@ -0,0 +1,41 @@
+from contextlib import contextmanager
+
+
+class ModifierException(Exception):
+    """Base Exception for Modifier error"""
+
+
+class Modifier(object):
```

Comment:
If not, then can you use the ABC module and mark the functions that need to be implemented with the @ abstractmethod. Details here: https://docs.python.org/2/library/abc.html